lib/modules/manager/npm/update/dependency/index.ts

```diff
@@ -1,12 +1,28 @@
 import { dequal } from 'dequal';
-import type { PackageJson } from 'type-fest';
 import { logger } from '../../../../../logger';
 import { escapeRegExp, regEx } from '../../../../../util/regex';
 import { matchAt, replaceAt } from '../../../../../util/string';
 import type { UpdateDependencyConfig } from '../../../types';
+import type { DependenciesMeta, NpmPackage } from '../../extract/types';
+
+function renameObjKey(
+  oldObj: DependenciesMeta,
+  oldKey: string,
+  newKey: string
+): DependenciesMeta {
+  const keys = Object.keys(oldObj);
+  return keys.reduce((acc, val) => {
+    if (val === oldKey) {
+      acc[newKey] = oldObj[oldKey];
+    } else {
+      acc[val] = oldObj[val];
```

Comment:
```suggestion
  return keys.reduce((acc, key) => {
    if (key=== oldKey) {
      acc[newKey] = oldObj[oldKey];
    } else {
      acc[key] = oldObj[key];
```

So it's more clear that it's the property name and not the  value.